src/test/java/rueppellii/backend2/tribes/resource/ResourceServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package rueppellii.backend2.tribes.resource;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+public class ResourceServiceTest {
+
+    private ResourceService resourceService;
+    private Resource resource;
+    private ResponseEntity r;
```

Comment:
I think in some case "response" would be more readable than "r"

for example:

.... .isEqualTo(response);  (like a book 😄)